src/stdfblib/ita/FakeTimeDev.h

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2005, 2024 ACIN, Profactor GmbH, fortiss GmbH,
+ *                                Primetals Technologies Austria GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Alois Zoitl, Thomas Strasser, Gerhard Ebenhofer, Rene Smodic, Ingo Hegny
+ *               - initial API and implementation and/or initial documentation
+ *   Alois Zoitl - Copied and modified from RMT_DEV.h
+ *******************************************************************************/
+#pragma once
+
+#include <device.h>
+#include <if2indco.h>
```

Comment:
Please use `""` to include local headers.

Reply:
Done